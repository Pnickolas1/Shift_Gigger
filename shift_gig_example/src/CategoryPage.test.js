import React from 'react';
import CategoryPage from './components/Category/CategoryPage'
import renderer from 'react-test-renderer';
import configureStore from 'redux-mock-store';
import { shallow } from 'enzyme'
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });



// create any initial state needed
const initialState = {}; 
// here it is possible to pass in any middleware if needed into //configureStore
const mockStore = configureStore();
let wrapper;
let store;
beforeEach(() => {
  //creates the store with any initial state or middleware needed  
  store = mockStore(initialState)
  wrapper = shallow(<CategoryPage store={store}/>)
 })


it('Jest - CategoryPage snapshop renders correctly', () => {
  const tree = renderer
    .create(<wrapper/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});

