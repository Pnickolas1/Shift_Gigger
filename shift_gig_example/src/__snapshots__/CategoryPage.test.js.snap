// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Jest - CategoryPage snapshop renders correctly 1`] = `<wrapper />`;
